pkgs/top-level/python-packages.nix

```diff
@@ -4229,6 +4229,8 @@ in {
 
   lsassy = callPackage ../development/python-modules/lsassy { };
 
+  luddite = callPackage ../development/python-modules/luddite { };
```

Comment:
Can this be used as a library? If not we don't need this entry.

Reply:
Yes this can be used as a library